travis-build/Makefile

```diff
@@ -67,7 +69,7 @@ prepare_brew:
 prepare_osx: ${LOCAL_DIR}/lib/librocksdb.dylib ${LOCAL_DIR}/bin/pd-server prepare-rustfmt | prepare_brew prepare_rust
 
 test_linux test_osx:
-	./travis-build/test.sh
+	export CI=true && ${CI_BUILD_DIR}/test.sh
```

Comment:
CI=true ${CI_BUILD_DIR}/test.sh

when this variable is not used by other scripts.